cmake/CppInterOp/CppInterOpConfig.cmake.in

```diff
@@ -10,19 +10,20 @@ get_filename_component(CPPINTEROP_INSTALL_PREFIX "${CPPINTEROP_INSTALL_PREFIX}"
 include(CMakeSystemSpecificInformation)
 
 ### build/install workaround
+if (@BUILD_SHARED_LIBS@ STREQUAL "ON")
```

Comment:
@vgvassilev The build of xeus-clang-repl fails if I use the if (BUILD_SHARED_LIBS) you suggested, but passes if i use if (@BUILD_SHARED_LIBS@ STREQUAL "ON") . Do you have any objections to me using this line instead?

Reply:
Ah, I see. I'd not hardcode `BUILD_SHARED_LIBS` to `ON`. It can be `TRUE`, `1`, etc. Doesn't this work:
```suggestion
if (@BUILD_SHARED_LIBS@)
```

In addition we need to fix this line `add_library(clangCppInterOp SHARED IMPORTED)` to `STATIC` conditionally.